<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_Team</class>
 <widget class="QMainWindow" name="New_Team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New_Team</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addteam">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="Add_stats_lable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Stats</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="player_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>player</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="metches_inp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>matches</string>
    </property>
   </widget>
   <widget class="QLabel" name="Add_team_data_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Team Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="runs_inp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>runs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hundreds_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>100s</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fifties_inp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>50s</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_inp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ctg_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>142</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ctg</string>
    </property>
   </widget>
   <widget class="QLabel" name="Add_match_lable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addstats_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add stats</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_match_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>players</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_scored_inp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>scored</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_faced_inp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>460</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Faced</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_fours_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Fours</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_sixes_inp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Sixes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_bowled_inp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Bowled</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_wkts_inp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Wkts</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_given_inp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Given</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_maiden_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>540</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Maiden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_stumping_inp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>580</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Stumping</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_ro_inp">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>580</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>RO*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMatch_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add match</string>
    </property>
   </widget>
   <widget class="QLabel" name="is_data_added">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(26, 95, 180)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="players_catches_inp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>580</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>catches</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>233</width>
      <height>143</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Players">
         <property name="text">
          <string>players</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="players"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="TeamName">
         <property name="text">
          <string>Team Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="teamname"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Value">
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
